aiogram/webhook/aiohttp_server.py

```diff
@@ -184,6 +186,10 @@ async def _handle_request(self, bot: Bot, request: web.Request) -> web.Response:
         return web.Response(body=self._build_response_writer(bot=bot, result=result))
 
     async def handle(self, request: web.Request) -> web.Response:
+        if self.secret_token:
+            t_token = request.headers.get("X-Telegram-Bot-Api-Secret-Token")
```

Comment:
What does mean prefix `t_`?

I any short prefixes is used in project it should be conventioned in the contribution guide, but as you see we doesn't have any prefixes convention in the project.

So, please set correct and understandable name for the variable.